app/ante/evm/eth_benchmark_test.go

```diff
@@ -66,10 +63,28 @@ func BenchmarkEthGasConsumeDecorator(b *testing.B) {
 				vmdb = testutil.NewStateDB(cacheCtx, s.app.EvmKeeper)
 				cacheCtx = s.prepareAccount(cacheCtx, addr.Bytes(), tc.balance, tc.rewards)
 				s.Require().NoError(vmdb.Commit())
+				keepers := ethante.ConsumeGasKeepers{
+					Bank:         s.app.BankKeeper,
+					Distribution: s.app.DistrKeeper,
+					Evm:          s.app.EvmKeeper,
+					Staking:      s.app.StakingKeeper,
+				}
+
+				baseFee := s.app.FeeMarketKeeper.GetParams(s.ctx).BaseFee
+				fee := tx.GetEffectiveFee(baseFee.BigInt())
+				denom := s.app.EvmKeeper.GetParams(s.ctx).EvmDenom
+				fees := sdk.NewCoins(sdk.NewCoin(denom, sdk.NewIntFromBigInt(fee)))
+				bechAddr := sdk.AccAddress(addr.Bytes())
 
 				// Benchmark only the ante handler logic - start the timer
 				b.StartTimer()
-				_, err := dec.AnteHandle(cacheCtx.WithIsCheckTx(true).WithGasMeter(sdk.NewInfiniteGasMeter()), tx, false, testutil.NextFn)
+
+				err := ethante.ConsumeFeesAndEmitEvent(
```

Comment:
Why the benchmark change?

Reply:
Because the legacy AnteHander was removed so I needed to adapt it to the new function!